packages/core/src/dispatcher.ts

```diff
@@ -58,8 +62,70 @@ export default class Dispatcher extends EventEmitter {
    */
   async init(): Promise<void> {
     this._peerId = this._peerId || (await this._ipfs.id()).id
-    await this._ipfs.pubsub.subscribe(this.topic, this.handleMessage.bind(this))
-    this._log({ peer: this._peerId, event: 'subscribed', topic: this.topic })
+    await this._subscribe()
+    !TESTING && this._resubscribeOnDisconnect()
```

Comment:
Can we just make this an if-statement?  Just eliminates some cognitive load in interpreting these inline expressions IMO